src/Endpoints/CertificateManagers.php

```diff
@@ -0,0 +1,205 @@
+use Cyberfusion\ClusterApi\Support\ListFilter;
+
+class CertificateManagers extends Endpoint
+{
+    /**
+     * @throws RequestException
+     */
+    public function list(ListFilter $filter = null): Response
+    {
+        if (is_null($filter)) {
+            $filter = new ListFilter();
+        }
+
+        $request = (new Request())
+            ->setMethod(Request::METHOD_GET)
+            ->setUrl(sprintf('certificate-managers?%s', $filter->toQuery()));
+
+        $response = $this
+            ->client
+            ->request($request);
+        if (!$response->isSuccess()) {
+            return $response;
+        }
+
+        return $response->setData([
+            'certificateManagers' => array_map(
+                function (array $data) {
+                    return (new CertificateManager())->fromArray($data);
+                },
+                $response->getData()
+            ),
+        ]);
+    }
+
+    /**
+     * @throws RequestException
+     */
+    public function get(int $id): Response
+    {
+        $request = (new Request())
+            ->setMethod(Request::METHOD_GET)
+            ->setUrl(sprintf('certificate-managers/%d', $id));
+
+        $response = $this
+            ->client
+            ->request($request);
+        if (!$response->isSuccess()) {
+            return $response;
+        }
+
+        return $response->setData([
+            'certificateManager' => (new CertificateManager())->fromArray($response->getData()),
+        ]);
+    }
+
+    /**
+     * @throws RequestException
+     */
+    public function create(CertificateManager $certificateManager): Response
+    {
+        $this->validateRequired($certificateManager, 'create', [
+            'common_names',
+            'provider_name',
+            'request_callback_url',
+            'cluster_id',
+        ]);
+
+        $request = (new Request())
+            ->setMethod(Request::METHOD_POST)
+            ->setUrl('certificate-managers')
+            ->setBody($this->filterFields($certificateManager->toArray(), [
+                'common_names',
+                'provider_name',
+                'request_callback_url',
+                'cluster_id',
+            ]));
+
+        $response = $this
+            ->client
+            ->request($request);
+        if (!$response->isSuccess()) {
+            return $response;
+        }
+
+        $certificateManager = (new CertificateManager())->fromArray($response->getData());
+
+        // Log which cluster is affected by this change
+        $this
+            ->client
+            ->addAffectedCluster($certificateManager->getClusterId());
+
+        return $response->setData([
+            'certificateManager' => $certificateManager,
+        ]);
+    }
+
+    /**
+     * @param CertificateManager $certificateManager
+     * @return Response
+     * @throws RequestException
+     */
+    public function update(CertificateManager $certificateManager): Response
```

Comment:
The docs aren't listing an update endpoint. Is this a copy-paste mistake or are the docs not completely up-to-date?

Reply:
The endpoint exists. The documentation will be updated in the next version.